app/src/main/java/org/glucosio/android/DomainConstants.java

```diff
@@ -0,0 +1,5 @@
+package org.glucosio.android;
+
+public class DomainConstants {
+    public static final String MG_D_L = "mg/dL";
```

Comment:
Reading in a file `MG_D_L` doesn't tell the developer what it means, could use a name explaining the meaning of this, for example, `DEFAULT_MEASUREMENT_UNIT` or just `MEASUREMENT_UNIT`, so if (by somehow) change the unit you don't need to refactor the name.

Reply:
`MG_D_L`?   Shouldn't be there `MG_DL`? Yes, this name would may not be not too specific to programmer, if he is not into diabetic issues. Think about this structure: https://gist.github.com/piotrek1543/1fb86c5089ad00d51f6c or try to implement nested enums

Then you would use `textView.setText([Constants.Units.MG_GL)` or `textView.setText(Constants.UNITS.MG_GL)` :-)